### ISSUE TITLE
Hidden Reply Boosting Problem

### ISSUE THREAD

**Issue by ericvolp12** [bluesky-social/proposals#7]
On Twitter, hidden replies end up being boosted inadvertently because they're "special" and have a separate button to view them (granted, normally near the top of the screen). I wonder how the hidden reply feature might be implemented in a way that doesn't inadvertently boost replies that the thread moderator marks for hiding.

**Comment by pfrazee** [comment]
This is the kind of inevitable tradeoff of hiding versus removing. Hiding supports more accountability but the urge to look at the hidden replies is pretty enormous.

Maybe it's just a UI thing? If it was buried behind a menu, there wouldnt be any tantalizing button to click?

**Comment by ericvolp12** [comment]
Yeah, I think limiting visibility too might help i.e. only allow showing a hidden reply to the person who hid them or to the person whose reply was hidden. Prevents a "harassment leaderboard" from forming and antipatterns developing that encourage bad behavior.

**Comment by Jacob2161** [comment]
This seems worth considering but it would surprising to find that there was a sustained boosting effect, in general and over time. Because as a rule having any kind of friction (like having to press "Show hidden replies") almost always drastically reduces the numbers of users that bother.

It should function a lot like the "spam" folder in Gmail: users will certainly check it out at first (and occasionally) but over time will come to learn that it's almost never worth looking at. Most importantly, thread creators will not feel pressure to respond to hidden replies.

On this topic it's particularly important to keep in mind that (due to decentralization) any reply "hiding" or "deleting" of posts from another user's repo will always be advisory. This means that there's only so much that can be reasonably mandated without (legit/responsible) clients deciding to offer their users more transparency around thread moderation.

Client preferences/defaults do seem particularly powerful here. It seems reasonable that clients might not even display the "Show hidden replies button" by default.
